component/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "component"
+version = "0.0.0"
+edition = "2021"
```

Comment:
We'll need the rest of the metadata if we're to publish future `fuelup` versions to crates.io.